Radio.m

```diff
@@ -808,25 +815,34 @@ - (void) initializeRadio {
         [self commandToRadio:[NSString stringWithFormat:@"client udpport %i", (int)self.vitaManager.vitaPort]];
     
     [self commandToRadio:@"keepalive enable"];
+
     [self commandToRadio:@"sub tx all"];
     [self commandToRadio:@"sub atu all"];
     [self commandToRadio:@"sub meter all"];
-    [self commandToRadio:@"sub slice all"];
     [self commandToRadio:@"sub pan all"];
-    [self commandToRadio:@"eq rx info"];
-    [self commandToRadio:@"eq tx info"];
+    [self commandToRadio:@"sub slice all"];
+    [self commandToRadio:@"sub gps all"];
```

Comment:
Can we take this out?  The GPS updates are a royal PITA.  I think we need the GPS support as an independent set of calls - we can add this back in at that time.

BTW - I think I am going to redo the logging in all these clases using CocoaLumberjack - its a much better mousetrap than NSLog with different levels and syntax identical to NSLog - that will help make it less cluttered in the logging - oh and Lumberjack supports logging to file too... useful for field support! :-)